src/app/components/Item/ItemsDefinitionLists.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { isArray as _isArray, isEmpty as _isEmpty } from 'underscore';
+import StatusLinks from './StatusLinks';
+import appConfig from '../../data/appConfig'
+
+const ItemsDefinitionLists = ({ items, holdings, bibId, id, searchKeywords, page }) => {
+
+  if (
+    !_isArray(items) ||
+    !items.length ||
+    items.every(item => item.isElectronicResource)
+  ) {
+    return null;
+  }
+
+  return (
+    <div key={`${bibId}-items`} id={`${bibId}-items`} className="items-definition-list">
+      {
+        items.map(item => {
+
+            if (_isEmpty(item)) {
+              return null;
+            }
+
+            if (item.isElectronicResource) {
+              return null;
+            }
+
+            let itemLocation;
+            if (item.location && item.locationUrl) {
+              itemLocation = (
+                <a href={item.locationUrl} className="itemLocationLink">{item.location}</a>
```

Comment:
i think it was decided in stand up that the item location is no longer going to be a link